HLTriggerOffline/Egamma/python/HLTmultiTrackValidatorGsfTracks_cff.py

```diff
@@ -0,0 +1,54 @@
+import FWCore.ParameterSet.Config as cms
+
+from Validation.RecoTrack.HLTmultiTrackValidator_cfi import *
+hltGsfTrackValidator = hltMultiTrackValidator.clone(
+    label = [
+        "hltEgammaGsfTracks",
+        "hltEgammaGsfTracksUnseeded",
+    ],
+    label_tp_effic           = "trackingParticlesElectron",
+    label_tp_effic_refvector = cms.bool(True), 
+    dirName                  = cms.string('HLT/EG/Tracking/ValidationWRTtp/'),
+)
+hltGsfTrackValidator.histoProducerAlgoBlock.GpSelectorForEfficiencyVsEta.maxRapidity = cms.double( 3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.GpSelectorForEfficiencyVsEta.minRapidity = cms.double(-3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.GpSelectorForEfficiencyVsPhi.maxRapidity = cms.double( 3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.GpSelectorForEfficiencyVsPhi.minRapidity = cms.double(-3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.GpSelectorForEfficiencyVsPt.maxRapidity = cms.double( 3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.GpSelectorForEfficiencyVsPt.minRapidity = cms.double(-3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.GpSelectorForEfficiencyVsVTXR.maxRapidity = cms.double( 3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.GpSelectorForEfficiencyVsVTXR.minRapidity = cms.double(-3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.GpSelectorForEfficiencyVsVTXZ.maxRapidity = cms.double( 3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.GpSelectorForEfficiencyVsVTXZ.minRapidity = cms.double(-3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.TpSelectorForEfficiencyVsEta.maxRapidity = cms.double( 3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.TpSelectorForEfficiencyVsEta.minRapidity = cms.double(-3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.TpSelectorForEfficiencyVsPhi.maxRapidity = cms.double( 3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.TpSelectorForEfficiencyVsPhi.minRapidity = cms.double(-3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.TpSelectorForEfficiencyVsPt.maxRapidity = cms.double( 3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.TpSelectorForEfficiencyVsPt.minRapidity = cms.double(-3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.TpSelectorForEfficiencyVsVTXR.maxRapidity = cms.double( 3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.TpSelectorForEfficiencyVsVTXR.minRapidity = cms.double(-3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.TpSelectorForEfficiencyVsVTXZ.maxRapidity = cms.double( 3.0)
+hltGsfTrackValidator.histoProducerAlgoBlock.TpSelectorForEfficiencyVsVTXZ.minRapidity = cms.double(-3.0)
```

Comment:
The eta limits for all the `TpSelector*`'s are already controlled with eras (i.e. set to +-3 for phase1) in
https://github.com/cms-sw/cmssw/blob/CMSSW_9_0_X/Validation/RecoTrack/python/TrackingParticleSelectionsForEfficiency_cff.py
Please continue using those (if they don't get propagated here for some reason, that should be fixed instead)

The `GpSelector*` are not, but the `MultiTrackValidatorGenPs` is probably out-of-date anyway (and not used in standard workflows).